Yank/analyze.py

```diff
@@ -848,7 +848,9 @@ class MultiExperimentAnalyzer(object):
     def __init__(self, script, **builder_kwargs):
         self.script = script
         self.builder = ExperimentBuilder(script=script, **builder_kwargs)
-        self.paths = self.builder.get_experiment_directories()
+        # Trap bug where paths ending in the separator cause errors later, ensure they don't end in it.
+        self.paths = tuple(path[:-1] if path[-1] == os.sep else path
+                           for path in self.builder.get_experiment_directories())
```

Comment:
It may be worth normalizing the paths directly in `ExperimentBuilder.get_experiment_directories()` or maybe in `ExperimentBuilder._get_experiment_dir()` with `os.path.normpath()` instead, so that nobody using that function will stumble into the same problem. That function should always return a path without a trailing `/`.

Reply:
`normpath` is a way better solution. ading it now